doc/admin/pool_operations.md

```diff
@@ -147,7 +139,7 @@ $ dmg pool get-acl --pool <UUID>
 ```
 
 The output is in the same string format used in the ACL file during creation,
-with one ACE per line.
+with one Access Control Entry (i.e. ACE) per line.
```

Comment:
/doc/user/container.md probably needs to be updated with similar changes for consistency, as the container version of the ACL command docs has much of the same wording.